homeassistant/components/isy994/strings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "Universal Devices ISY994",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "host": "URL",
+          "password": "[%key:common::config_flow::data::password%]",
+          "tls": "The TLS version of the ISY controller."
+        },
+        "description": "The host entry must be in full URL format, e.g., http://192.168.10.100:80",
+        "title": "Connect to your ISY994"
+      }
+    },
+    "error": {
+      "unknown": "[%key:common::config_flow::error::unknown%",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "invalid_host": "The host entry was not in full URL format, e.g., http://192.168.10.100:80"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%"
```

Comment:
Bad ending, it must have a `]` at the end.

Reply:
Thanks @McGiverGim

Will note in https://github.com/home-assistant/core/pull/35467